### PR TITLE
feat(fal): introduce fal doctor

### PR DESCRIPTION
**Pull request by efiop**
A place to have all of our useful info that one might need.

Screenshot 2024-06-14 at 02 47 49

### CODE REVIEW

File: projects/fal/src/fal/cli/doctor.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    table.add_row("isolate", isolate.__version__)
+
+    table.add_row("", "")
+    table.add_row("python", platform.python_version())
+    table.add_row("platform", platform.platform())
+
+    table.add_row("", "")
+    table.add_row("FAL_HOST", fal.flags.GRPC_HOST)
+    FAL_KEY = os.getenv("FAL_KEY", "")
+    table.add_row("FAL_KEY", (FAL_KEY[:9] + "...") if FAL_KEY else "")
```
Comment (chamini2) [inline projects/fal/src/fal/cli/doctor.py:25]: maybe show the entire left part of the `:` so it is easy to id it for us

**Review by chamini2 — APPROVED** [review]
good idea

**Review by kakkoyun — COMMENTED** [review]
LGTM except the name maybe. It looks like an `info` or `system-info` command to me. The usual expectation from the `doctor` command is to check required prerequisite or setup to be in place. It would either say "everything looks ok", or "fix ..." e.g. `brew doctor`, `pyenv docktor`.

**Comment by efiop** [comment]
@kakkoyun `info` is too generic, `system-info` also not very common. `fal doctor` can in the future check for stuff and print warnings, it is meant to be generic place for debugging/helpful info and checks. Had the same naming struggle with my previous project too, but doctor seemed to stick around and never was a problem, so for a fal cli it is probably also fine for now. We can change it up though, no compatibility promises here or anything.

File: projects/fal/src/fal/cli/doctor.py (hunk 1 above)
Comment (efiop) [reply projects/fal/src/fal/cli/doctor.py:25]: Forgot to push, so had to create a followup https://github.com/fal-ai/fal/pull/242

**Pull request merged**